components/HTTP/src/Websocket.cpp

```diff
@@ -56,6 +56,8 @@ esp_err_t bytesToHex(std::uint8_t* buf, char* dst, std::size_t len)
         dst[i * 2 + 1] = hexmap[lower];
     }
 
+    std::cout << "dios meo" << dst << std::endl;
```

Comment:
```suggestion
    ESP_LOGW(TAG, "why this doens't work: %s!!!", dst);
```

Reply:
this was resolved in the commit 2dd4a1f9fa14aebe92dea0cfc885e000929295c0